Libraries/Opc.Ua.Server/Session/Session.cs

```diff
@@ -391,6 +391,11 @@ public string SecureChannelId
             }
         }
 
+        /// <summary>
+        /// allow derived classes access
+        /// </summary>
+        protected int MaxBrowseContinuationPoints { get => m_maxBrowseContinuationPoints; set => m_maxBrowseContinuationPoints = value; }
```

Comment:
make the property public and add proper description?

Reply:
I'm a bit afraid of doing that. If I perceive this correctly. these server capabilities are read from the configuration and then  passed on to various classes. Making this public may lead people to use it and change the value with unexpected results. For the unit tests I could confirm that it does what I want.

Reply:
ok